README.md

```diff
@@ -134,8 +134,8 @@ module.
 Gotchas [↑](#structure-of-this-tutorial)
 -------
 
-* `head`, `tail`, `last`, `init` work with `NonEmpty a` instead of `[a]`.
-* Safe analogue for `head` function: `safeHead :: [a] -> Maybe a`.
+* `head`, `tail`, `last`, `init`, `foldl1`, `minimum` and other were-partial functions work with `NonEmpty a` instead of `[a]`.
+* Safe analogue for `head`, `foldl1`, `minimum` and functions, for instance: `safeHead :: [a] -> Maybe a`.
```

Comment:
```suggestion
* Safe analogue for `head`, `foldl1`, `minimum` and `maximum` functions, for instance: `safeHead :: [a] -> Maybe a`.
```
I think you forgot that one

Reply:
I was lazy to enlist all functions, but probably it's really worth doing :+1:

Reply:
Oh I agree it is not necessary, but I supposed you missed one only because of the `and functions` (there was the `and` already but nothing followed it before `functions`).

Reply:
Ah got it, there was really a typo there.

Okay, probably both options seem fine, let it be as it is now.